server/constants/constants.go

```diff
@@ -92,12 +92,15 @@ const (
 	QueryParamPerPage     = "per_page"
 
 	// Filters
-	FilterCreatedByMe     = "me"
-	FilterCreatedByAnyone = "anyone"
-	FilterAllChannels     = "all_channels"
-	FilterAll             = "all"
-	FilterBoards          = "boards"
-	FilterRepos           = "repos"
+	FilterCreatedByMe            = "me"
+	FilterCreatedByAnyone        = "anyone"
+	FilterAllChannels            = "all_channels"
+	FilterAll                    = "all"
+	FilterBoards                 = "boards"
+	FilterRepos                  = "repos"
+	EventTypeRelease             = "release"
+	FilterIDReleaseDefinitionID  = "releaseDefinitionId"
```

Comment:
Is 2 times ID is needed in these variables ?

Reply:
Yes, the first part is prefix and the second one is the name

Reply:
ok then i think the last ID is not needed like for the filter for boards we have filterBoards not filterBoardsFilter

Reply:
Done